auth/src/main/res/layout/welcome_back_idp_prompt_layout.xml

```diff
@@ -23,6 +23,7 @@
 
     <Button
         android:id="@+id/welcome_back_idp_button"
+        style="@style/FirebaseUI.Button"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_gravity="end"
```

Comment:
Since we are adding the style, we can remove all of the following:
```xml
android:layout_width="wrap_content"
android:layout_height="wrap_content"
android:layout_gravity="end"
```